### ISSUE TITLE
Feature request: Carthage support

### ISSUE THREAD

**Issue by mortenbekditlevsen** [iZettle/sdk-ios#139]
Would it at all be possible to support Carthage as dependency resolver alongside with cocoapods?

**Comment by arikis** [comment]
@mortenbekditlevsen we're looking into Carthage for the next SDK release.

**Comment by mortenbekditlevsen** [comment]
Saw that there was a release today - did Carthage support make it into 1.5.0?

**Comment by arikis** [comment]
Hi @mortenbekditlevsen,

Experimental support for Carthage has now been added. You can try adding following to your Cartfile

```
binary "https://raw.githubusercontent.com/iZettle/sdk-ios/master/carthage.json" >= 1.5
```

More info in README will be coming soon.

**Comment by mortenbekditlevsen** [comment]
Hi @arikis ,
Thank you very much - this works perfectly!
One minor request for a change:
Carthage appears to be using the base file name as the reference to the framework, so right now it is called 'carthage'. For instance if I only want to update this single framework I would do:
carthage update <options> carthage
Would it at all be possible to have the json file renamed to iZettle.json (and then possibly placed in a 'carthage' subfolder if you wish to make it clear that it's a carthage thing only)?

**Comment by arikis** [comment]
I’m glad it works for you! That makes sense, I’ll open up a PR later today.

**Comment by arikis** [comment]
@mortenbekditlevsen, the changes are now implemented (see #160). You'll need to update your Cartfile to:

```
binary "https://raw.githubusercontent.com/iZettle/sdk-ios/master/Carthage/iZettleSDK.json" >= 1.5
```
